src/dune_engine/scheduler.ml

```diff
@@ -57,7 +57,9 @@ module Shutdown = struct
       | _ -> None)
 end
 
-let blocked_signals : Signal.t list = [ Int; Quit; Term ]
+let interrupt_signals : Signal.t list = [ Int; Quit; Term ]
+
+let blocked_signals : Signal.t list = Winch :: interrupt_signals
```

Comment:
Could you add some comments on the difference between `interrupt_signals` and `blocked_signals`?